src/Compilers/CSharp/Test/Symbol/Symbols/FunctionPointerTypeSymbolTests.cs

```diff
@@ -268,36 +282,61 @@ class C
 }");
 
             comp.VerifyDiagnostics(
+                // (4,47): error CS9502: The target runtime doesn't support extensible unmanaged calling conventions.
+                //     public unsafe void M1(delegate* unmanaged[invalid]<void> p) {}
+                Diagnostic(ErrorCode.ERR_RuntimeDoesNotSupportUnmanagedExtensionCallConv, "invalid").WithLocation(4, 47),
                 // (4,47): error CS8807: 'invalid' is not a valid calling convention specifier for a function pointer. Valid conventions are 'Cdecl', 'Stdcall', 'Thiscall', and 'Fastcall'.
```

Comment:
>Valid conventions are 'Cdecl', 'Stdcall', 'Thiscall', and 'Fastcall' [](start = 119, length = 68)

This feels misleading.

Reply:
How so?

---
In reply to: [457765833](https://github.com/dotnet/roslyn/pull/46110#discussion_r457765833) [](ancestors = 457765833)

Reply:
> How so?

In a sense is that any identifier is a fair game if we can locate the corresponding type. So, the closed set of valid conventions is misleading in my opinion.

---
In reply to: [457768821](https://github.com/dotnet/roslyn/pull/46110#discussion_r457768821) [](ancestors = 457768821,457765833)